src/lib/stores/advanced-visibility.ts

```diff
@@ -1,17 +1,17 @@
 import { derived } from 'svelte/store';
-import { page } from '$app/stores';
 
 import { cluster } from './cluster';
 import { temporalVersion } from './versions';
+import { settings } from './settings';
 
 import {
   advancedVisibilityEnabled,
   advancedVisibilityEnabledWithOrderBy,
 } from '$lib/utilities/advanced-visibility-enabled';
 
 export const isCloud = derived(
-  [page],
-  ([$page]) => $page.data?.settings?.runtimeEnvironment?.isCloud,
```

Comment:
I'm thinking we should remove the $settings store. I don't think we need it anymore now that we are setting the settings prop in cloud. Any reference to settings should come from $page.data.settings so we have one source of truth.